package/MDAnalysis/visualization/streamlines.py

```diff
@@ -45,22 +54,39 @@
         'http://matplotlib.org/faq/installing_faq.html?highlight=install')
 
 import MDAnalysis
-import multiprocessing
-import numpy as np
-import scipy
+
 
 
 def produce_grid(tuple_of_limits, grid_spacing):
-    '''Produce a grid for the simulation system based on the tuple of Cartesian Coordinate limits calculated in an
-    earlier step.'''
+    """Produce a grid for the simulation system based on the tuple of Cartesian Coordinate limits calculated in an
+    earlier step."""
     x_min, x_max, y_min, y_max = tuple_of_limits
     grid = np.mgrid[x_min:x_max:grid_spacing, y_min:y_max:grid_spacing]
     return grid
 
 
 def split_grid(grid, num_cores):
-    '''Take the overall grid for the system and split it into lists of square vertices that can be distributed to
-    each core. Limited to 2D for now'''
+    """Split the grid into blocks of vertices.
+
+    Take the overall `grid` for the system and split it into lists of
+    square vertices that can be distributed to each core.
+
+    Parameters
+    ----------
+    grid : numpy.array
+        2D array
+    num_cores : int
+        number of partitions to generate
+
+    Returns
+    -------
+    [list_square_vertex_arrays_per_core, list_parent_index_values, current_row, current_column]
```

Comment:
Sorry, not sure what you mean "should be split into a list like below in the file change view". "like below"??